include/internal/libfreenect2/rgb_packet_processor.h

```diff
@@ -108,5 +109,22 @@ class VTRgbPacketProcessor : public RgbPacketProcessor
 };
 #endif
 
+#ifdef LIBFREENECT2_WITH_VAAPI_SUPPORT
+class VaapiRgbPacketProcessorImpl;
+
+class VaapiRgbPacketProcessor : public RgbPacketProcessor
+{
+public:
+  VaapiRgbPacketProcessor();
+  virtual ~VaapiRgbPacketProcessor();
+  virtual bool good();
+  virtual Allocator *getAllocator();
+protected:
+  virtual void process(const libfreenect2::RgbPacket &packet);
```

Comment:
I don't know. I was just copying the definitions. Is there originally some intention to disallow directly access to the packet processor implementations outside of the async wrapper?